src/Layer/TiledGeometryLayer.js

```diff
@@ -105,22 +139,114 @@ class TiledGeometryLayer extends GeometryLayer {
         }
     }
 
+    /**
+     * Update a node of this layer. The node will not be updated if:
+     * <ul>
+     *  <li>it does not have a parent, then it is removed</li>
+     *  <li>its parent is being subdivided</li>
+     *  <li>is not visible in the camera</li>
+     * </ul>
+     *
+     * @param {Object} context - The context of the update; see the {@link
+     * MainLoop} for more informations.
+     * @param {Layer} layer - Parameter to be removed once all update methods
+     * have been aligned.
+     * @param {TileMesh} node - The node to update.
+     *
+     * @returns {Object}
+     */
     update(context, layer, node) {
-        return processTiledGeometryNode(this.culling, this.subdivision)(context, this, node);
-    }
+        if (!node.parent) {
+            return ObjectRemovalHelper.removeChildrenAndCleanup(layer, node);
+        }
+        // early exit if parent' subdivision is in progress
+        if (node.parent.pendingSubdivision) {
+            node.visible = false;
+            node.setDisplayed(false);
+            return undefined;
+        }
 
-    onTileCreated(node) {
-        node.material.setLightingOn(this.lighting.enable);
-        node.material.uniforms.lightPosition.value = this.lighting.position;
+        // do proper culling
+        const isVisible = (!this.culling(node, context.camera));
```

Comment:
isVisible is uneless, could you use `node.visible` directly?

Reply:
Done